tools/vargo/src/main.rs

```diff
@@ -1061,6 +1084,23 @@ fn run() -> Result<(), String> {
 
                     std::fs::copy(&from_f, &to_f)
                         .map_err(|x| format!("could not copy file ({})", x))?;
+
+                    if is_exe {
+                        writeln!(
+                            &mut macos_prepare_script,
+                            "xattr -d com.apple.quarantine {}",
```

Comment:
While it is _probably_ fine (who uses directory names with spaces?!), it probably is a good idea for these file names (here and below, in all the `writeln`s) to have quoted file names)

Reply:
They are just file names (no other path segments) and they all don't have any special characters. I'm more worried about getting the quoting wrong, than to get a failure if we somehow introduce an artifact with a weird name.